examples/README.md

```diff
@@ -620,33 +617,6 @@ For this example the following is needed in the selected region:
 * Compute Engine API: Resource policies: **one for each job in parallel** -
   _only needed for `compute` partition_
 
-### [hpc-intel-select-slurm.yaml] ![community-badge]
-
-This example provisions a Slurm cluster automating the [steps to comply to the
-Intel Select Solutions for Simulation & Modeling Criteria][intelselect]. It is
-more extensively discussed in a dedicated [README for Intel
-examples][intel-examples-readme].
-
-[hpc-intel-select-slurm.yaml]: ../community/examples/intel/hpc-intel-select-slurm.yaml
-[intel-examples-readme]: ../community/examples/intel/README.md
-[intelselect]: https://cloud.google.com/compute/docs/instances/create-intel-select-solution-hpc-clusters
-
-### [pfs-daos.yaml] ![community-badge]
-
-This example provisions a DAOS cluster with [managed instance groups][migs] for the servers and for clients. It is more extensively discussed in a dedicated [README for Intel
-examples][intel-examples-readme].
-
-[pfs-daos.yaml]: ../community/examples/intel/pfs-daos.yaml
-[migs]: https://cloud.google.com/compute/docs/instance-groups
```

Comment:
Check that the `migs` link is not being used elsewhere.

Reply:
migs, doesn't have any other reference in this file.